Task_TheFlexGame/index.html

```diff
@@ -1,11 +1,43 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Flex Game</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for FLEX GAME TASK -->
-  </body>
-</html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <body style="margin: 0; padding: 0">
+           <div class="main">
+          <div class="header">Header</div>
+          <div class="content">
+            <div class="menu">
+            Menu
+            <div class="menuButton">1</div>
+            <div class="menuButton">2</div>
+            <div class="menuButton">3</div>
+          </div>
+            <div class="contentContainer">
+            Content
+            <div class="numDiv">
+            <div class="num">1</div>
+            <div class="num">2</div>
+            <div class="num">3</div>
```

Comment:
The opening and closing div tags should be on separate lines for better readability.